segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/metadata/StatusFlags.java

```diff
@@ -33,6 +33,11 @@ public final class StatusFlags {
      */
     public static final int SEALED = 0x0002;
 
+    /**
+     * Flag to indicate whether all writes are atomic.
+     */
+    public static final int ATOMIC_WRITES = 0x0004;
```

Comment:
Is this something encoded in the Storage Metadata information of each Segment? Could this break anything or is it using some unused space to encode more information for a Segment? Why using `0x0004` instead of `0x0003`, which looks like the next number to choose?

Reply:
It is a bit flag 
first bit -> is active or not   00001
second bit -> is sealed or not  00010
third bit -> is atomic or not  00100
fourth -> has ownership changed  01000 
fourth -> is system segment  10000 


we have 32 flags totals - only 5 used so far

Reply:
The status flag field is an integer and we use it as bitarray